src/app/components/Brand/index.jsx

```diff
@@ -3,22 +3,28 @@ import styled from 'styled-components';
 import {
   C_POSTBOX,
   C_WHITE,
-  GEL_SPACING_DBL,
+  GEL_SPACING_HLF,
+  GEL_SPACING,
+  GEL_SPACING_TRPL,
 } from '../../lib/constants/styles';
 
+// (( BANNER_HEIGHT - SVG_HEIGHT ) / 2 ) - A_PADDING_TOP
+const centerSvg = '20px';
+const bannerHeight = '80px';
+
 const StyledWrapper = styled.div`
   background-color: ${C_POSTBOX};
-  height: 80px;
+  height: ${bannerHeight};
   width: 100%;
-  padding: ${GEL_SPACING_DBL};
 `;
 
 const StyledLink = styled.a`
-  :hover,
-  :focus {
+  padding: ${GEL_SPACING} ${GEL_SPACING} 0;
+  display: inline-block;
+  :hover span,
```

Comment:
Nesting a `span` within the hover this way works, but it is not recommended for ensuring CSS is performant and robust.
It would be better to add a class to the generic styled span & reference that class here. 
e.g. 
```
:hover .brand-svg__wrapper,
```

```
const StyledSpan = styled.span.attrs({
  className: 'brand-svg__wrapper',
})`
  display: inline-block;
`;
```

I know that using classes and their naming conventions has not been discussed as a team, since we decided to use `styled-components`. It would be good to discuss this as a team. @bbc/news-articles

Reply:
After some discussion offline, I want to revise this comment. 
The change you'd need is to update the lines
```
:hover ${StyledSpan},
:focus ${StyledSpan},
```
and move the definition of `StyledSpan` to above the `StyledLink`.

Reference: https://www.styled-components.com/docs/advanced#referring-to-other-components